src/app/pages/charging-stations/details-component/charging-stations-start-transaction-details-dialog-component.ts

```diff
@@ -0,0 +1,243 @@
+import { StartTransaction } from 'app/types/Transaction';
+import { UserToken } from 'app/types/User';
+import { Utils } from 'app/utils/Utils';
+
+@Component({
+    templateUrl: './charging-stations-start-transaction-details-dialog-component.html',
+})
+export class ChargingStationsStartTransactionDetailsDialogComponent implements OnInit {
+    public title = '';
+    public chargeBoxID = '';
+    public formGroup!: FormGroup;
+    public user!: AbstractControl;
+    public userID!: AbstractControl;
+    public car!: AbstractControl;
+    public carID!: AbstractControl;
+    public tag!: AbstractControl;
+    public tagID!: AbstractControl;
+    private spinnerCounter = 0;
+
+    public loggedUser: UserToken;
+    public isAdmin = false;
+    constructor(
+        private dialog: MatDialog,
+        private router: Router,
+        public spinnerService: SpinnerService,
+        private messageService: MessageService,
+        private translateService: TranslateService,
+        private centralServerService: CentralServerService,
+        private authorizationService: AuthorizationService,
+        private dialogRef: MatDialogRef<ChargingStationsStartTransactionDetailsDialogComponent>,
+        @Inject(MAT_DIALOG_DATA) data: any) {
+        // Set
+        this.title = data.title;
+        this.chargeBoxID = data.chargeBoxID;
+        this.loggedUser = centralServerService.getLoggedUser();
+        this.isAdmin = this.authorizationService.isAdmin();
+        Utils.registerCloseKeyEvents(this.dialogRef);
+    }
+
+
+    public ngOnInit() {
+        // Init the form
+        this.formGroup = new FormGroup({
+            user: new FormControl('',
+                Validators.compose([
+                    Validators.required,
+                ])),
+            userID: new FormControl('',
+                Validators.compose([
+                    Validators.required,
+                ])),
+            car: new FormControl('',
+                Validators.compose([
+
+                ])),
+            carID: new FormControl('',
+                Validators.compose([
+                ])),
+            tag: new FormControl('',
+                Validators.compose([
+                    Validators.required,
+                ])),
+            tagID: new FormControl('',
+                Validators.compose([
+                    Validators.required,
+                ]))
+        });
+        // Form
+        this.user = this.formGroup.controls['user'];
+        this.userID = this.formGroup.controls['userID'];
+        this.car = this.formGroup.controls['car'];
+        this.carID = this.formGroup.controls['carID'];
+        this.tag = this.formGroup.controls['tag'];
+        this.tagID = this.formGroup.controls['tagID'];
+        this.user.setValue(Utils.buildUserFullName(this.loggedUser));
+        this.userID.setValue(this.loggedUser.id);
+        this.spinnerCounter = 2;
+        this.loadDefaultCar();
+        this.loadDefaultTag();
```

Comment:
You can also have one REST endpoint interface that will return both default car and badge (simplify front and backend).

We should call once the backend per UI displayed (Project Golden Rules # 15 violated :))

![image](https://user-images.githubusercontent.com/25266945/96595134-a5033f00-12eb-11eb-99b7-6336f8103b72.png)

Reply:
Done :)